### PR TITLE
1949 cardheader update

### PR DESCRIPTION
**Pull request by s-laugh**
### Description

List of proposed changes:

- Adds a required "callout" section to the card header
- Updates the design of the card header
- Updates the summaries of the card header to mach design (which elements are shown)

### What to test for/How to test

Validate that everything is working as expected, and no missing parts.

### Additional Notes

This sets what should be the correct padding/margin around the "apply" buttons, but the icon until updated makes it look too big.

### CODE REVIEW

File: constants/EIMappingCodes.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { TypeCodes } from './ProgramTypeCodes'
 import { StatusCodes } from './StatusCodes'
 
 export const EITypes = {
-  1: TypeCodes.EIUnknown,
+  1: TypeCodes.EISickness,
```
Comment (TishamIslam) [inline constants/EIMappingCodes.js:5]: Why do we have 2 TypeCodes with the same value?

**Comment by shewood** [comment]
You might want to consider sm:gap-2 or more so that the text for requested benefit doesn't touch the status update date with those massive fonts.

**Comment by s-laugh** [comment]
> You might want to consider sm:gap-2 or more so that the text for requested benefit doesn't touch the status update date with those massive fonts.

Great suggestion! I used a lg:gap-7 to get a 28px as the design grid shows gaps of ~30px; but also a sm:gap-4 because imo it was too much on smaller screens.

**Review by shewood — APPROVED** [review]
The gap change looks great.

**Pull request merged**